ci/install-dmd.ps1

```diff
@@ -0,0 +1,40 @@
+Set-StrictMode -Version latest
+$ErrorActionPreference = "Stop"
+$ProgressPreference = "SilentlyContinue"
+
+# default installation directory
+$dmd_install = "C:\D"
+
+#echo "Fetching latest DMD version..."
+$dmd_version = [System.Text.Encoding]::ASCII.GetString((Invoke-WebRequest -URI "http://downloads.dlang.org/releases/LATEST").content)
+$dmd_url = "http://downloads.dlang.org/releases/2.x/$dmd_version/dmd.$dmd_version.windows.zip"
+$dmd_filename = [System.IO.Path]::GetFileName($dmd_url)
+$dmd_archive = Join-Path ($env:temp) $dmd_filename
```

Comment:
I think you probably want to save this under `C:\cache` and fetch a specific version instead of the latest one?

Reply:
What's wrong with using the latest version? The script only stores the archive in system temp folder.

Reply:
Caching the output makes the CI more resilient to transient network failures (f.ex., if dlang.org is down). That's my main concern.